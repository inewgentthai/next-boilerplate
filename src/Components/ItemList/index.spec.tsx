// #region Global Imports
import * as React from "react";
// #endregion Global Imports

// #region Local Imports
import { render } from "@Test/utils";
import { ItemList } from "@Components";
// #endregion Local Imports

describe("ItemList", () => {
    it("should match snapshot", () => {
        const { container } = render(<ItemList>Test</ItemList>);

        expect(container).toMatchSnapshot();
    });
});
