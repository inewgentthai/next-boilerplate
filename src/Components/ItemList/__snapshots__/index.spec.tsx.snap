// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList should match snapshot 1`] = `
<div>
  <section
    class="item-list"
  >
    <div
      class="container"
    >
      <div
        class="row no-gutters"
      >
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              data-backdrop="static"
              data-target="#itemModal"
              data-toggle="modal"
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  L
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee1.png"
                  srcset="static/truecoffee-ordering/images/product-coffee1.png 2x, static/truecoffee-ordering/images/product-coffee1.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cafe Latte - LIced Cafe Latte - L
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee1.png"
                  srcset="static/truecoffee-ordering/images/product-coffee1.png 2x, static/truecoffee-ordering/images/product-coffee1.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Americano - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee3.png"
                  srcset="static/truecoffee-ordering/images/product-coffee3.png 2x, static/truecoffee-ordering/images/product-coffee3.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Caramel Macchiato - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  M
                </div>
                <img
                  class="product-thumb"
                  src="static/truecoffee-ordering/images/product-coffee4.png"
                  srcset="static/truecoffee-ordering/images/product-coffee4.png 2x, static/truecoffee-ordering/images/product-coffee4.png 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  Iced Cappuccino - M
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿85.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
